requirements.txt

```diff
@@ -0,0 +1 @@
+-r fv3core/requirements/requirements_container.txt
```

Comment:
I see this is appears... ah... opposite to what I said in the first review round.

What I mean to say is, if there is a `requirements.txt` at the top level, it should contain the requirements needed for the integration image. This could contain -r <subdir requrements> as you like in order to say the integration image depends on the subprojects, `-e fv3core` or something like that should also work assuming we keep the `setup.py` up to date with what is needed for the packages to run. But it probably shouldn't defer entirely to the fv3core requirements - the integration image is its own thing.

Reply:
Right, I had initially copied the `fv3core` requirements up one level, but it makes more sense that we don't keep two copies of requirements. Adding `-r fv3gfs-physics/<something>`. In the future, when integration has its own requirement, it will also be added.